multicluster/controllers/multicluster/member/serviceexport_controller.go

```diff
@@ -281,11 +289,16 @@ func (r *ServiceExportReconciler) Reconcile(ctx context.Context, req ctrl.Reques
 			eps.Subsets = []corev1.EndpointSubset{svcIPAsSubset}
```

Comment:
A few comments/questions for existing code:

1. I feel we should move GetServiceEndpointSubset, GetServiceEndpointPorts, FilterEndpointSubsets to this file, not in common, as they are not called by other packages(?).
2. GetServiceEndpointSubset should be called GetClusterIPEndpointSubset?
3. I saw GetServiceEndpointSubset adds all ClusterIPs to endpoint subsets. But when a Service can have multiple IPs? The dual stack case? If so, we should exclude the IPv6 ClusterIP?

Reply:
yes, moved these functions to this file and rename them. Add a IP type check to skip IPv6 ClusterIP. Thanks.